deeprankcore/query.py

```diff
@@ -414,11 +414,11 @@ def __init__(  # pylint: disable=too-many-arguments
             insertion_code (str): The insertion code of the variant residue, set to None if not applicable.
             wildtype_amino_acid (:class:`AminoAcid`): The wildtype amino acid.
             variant_amino_acid (:class:`AminoAcid`): The variant amino acid.
-            pssm_paths (Dict(str,str), optional): The paths to the .PSSM files, per chain identifier. Defaults to None.
-            radius (float, optional): In Ångström, determines how many residues will be included in the graph. Defaults to 10.0. 
-            distance_cutoff (float, optional): Max distance in Ångström between a pair of atoms to consider them as an external edge in the graph.
+            pssm_paths (Optional[Dict(str,str)], optional): The paths to the .PSSM files, per chain identifier. Defaults to None.
+            radius (Optional[float], optional): In Ångström, determines how many residues will be included in the graph. Defaults to 10.0. 
```

Comment:
Are we sure that radius can be None? Just check it to be sure :)

Reply:
I found that it cannot be None, I removed the Optional datat type.